GUIDE.md

```diff
@@ -0,0 +1,181 @@
+# Node Documentation
+
+Welcome to the Node.js Documentation Guide — or more colloquially, the docs on
```

Comment:
I doubt that many people actually know what colloquially means (myself included), if possible we should break it into more available terms.